icu4j/main/classes/core/src/com/ibm/icu/impl/ICUResourceBundle.java

```diff
@@ -1304,17 +1388,16 @@ public ICUResourceBundle load() {
 
             // fallback to locale ID parent
             if(b == null){
-                int i = localeName.lastIndexOf('_');
-                if (i != -1) {
-                    // Chop off the last underscore and the subtag after that.
-                    String temp = localeName.substring(0, i);
-                    b = instantiateBundle(baseName, temp, defaultID, root, openType);
+                String origLocaleName = (origLocaleID != null) ? origLocaleID : localeName;
+                String fallbackLocaleID = getParentLocaleID(localeName, origLocaleName);
+                if (fallbackLocaleID != null) {
```

Comment:
I am not seeing enough context here... If there is an explicit `%%Parent`, is that handled at the right time? Probably before the CLDR parent fallback?

Reply:
Unless I'm badly mistaken, all of this code is about locating the first resource bundle file that matches the requested locale ID and actually exists.  You don't get to `%%Parent` until you find an actual resource file, and at that point you're not in this code anymore.